apps/wpcom-block-editor/src/calypso/features/gutenboarding-editor-overrides.js

```diff
@@ -54,3 +58,65 @@ function updateSettingsBar() {
 		settingsBar.prepend( saveButton );
 	} );
 }
+
+function showFirstTimeNotice() {
+	const getPreviewButton = () => document.querySelector( '.editor-post-preview' );
+
+	const awaitPreviewButton = setInterval( () => {
+		const previewButton = getPreviewButton();
+		if ( ! previewButton ) {
+			return;
+		}
+		clearInterval( awaitPreviewButton );
+
+		// Create preview link
+		// Reuse the href from existing Preview button in the header
+		// and trigger the click event handler (registered through jQuery)
+		// as seen in handlePreview() in iframe-bridge-server.js.
+		const previewLink = (
+			<a
+				href={ previewButton.href }
```

Comment:
If we just use `href="javascript: void(0)"` here we won't need to use `awaitPreviewButton` anymore.

Reply:
Ignoring this.